administration/src/cards/PdfFactory.ts

```diff
@@ -18,6 +19,22 @@ export class PdfError extends Error {
   }
 }
 
+const loadCustomFontWithFallback = async (font: string, doc: PDFDocument, fallbackFont: string): Promise<PDFFont> => {
+  doc.registerFontkit(fontkit)
+  const fontUrl = `${process.env.PUBLIC_URL}/fonts/${font}`
+  const fontBytes = await fetch(fontUrl)
+    .then(res => {
+      if (res.ok && res.headers.get('Content-Type')?.includes('font')) {
+        return res.arrayBuffer()
+      }
+      reportError(`Couldn't load custom font ${font}. Using fallback font.`)
+    })
+    .catch(error => {
+      reportError(error)
+    })
+  return fontBytes ? doc.embedFont(fontBytes) : doc.embedFont(fallbackFont)
+}
```

Comment:
:wrench: Please don't mix async/await and then/catch syntax. I'd prefer if you'd just use async/await.

Reply:
good point. Adjusted it